activejob/CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
 
+- Add randomness to :exponentially_longer
```

Comment:
This leaves the reader with more questions than answers. Let's share the motivation, why it's needed, how it works, and what it means for your app.

Reply:
Gladly!